lib/datasource/gradle/index.js

```diff
@@ -0,0 +1,89 @@
+const got = require('got');
+const { coerce } = require('../../versioning/semver');
```

Comment:
I'm thinking whether we should just import `coerce` from the `semver` library directly rather than adding it to the interface of our `lib/versioning/semver` and adjusting those tests.